run_tests_cpu.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Force JAX to use only the CPU
+export JAX_PLATFORMS=cpu
+
+# Run pytest on the jfsd package
+pytest tests/test_class.py::TestClassCPU
```

Comment:
Does the following work? In that case you don't need to export the variable.
```suggestion
JAX_PLATFORMS=cpu pytest tests/test_class.py::TestClassCPU
```

Reply:
Done.